pkg/controller/replicaset/replica_set_utils.go

```diff
@@ -126,7 +126,7 @@ func calculateStatus(rs *extensions.ReplicaSet, filteredPods []*v1.Pod, manageRe
 	return newStatus
 }
 
-// NewReplicaSetCondition creates a new replica set condition.
+// NewReplicaSetCondition creates a new replicaset condition.
```

Comment:
Beauty is in the eye of the beholder. I like "replicaset" as one word better, so i'm going to lgtm this.